app/models/user.rb

```diff
@@ -56,6 +56,8 @@ class User < ApplicationRecord
   # The validation below has been commented out as it does not
   # work because :validatable in devise overrides this.
   # validates_uniqueness_of :email, scope: :account_id
+  attr_accessor :linked_account_id
```

Comment:
since we already have the relation to account via account users, this might end up being confusing in the future